koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -92,8 +92,8 @@ def __init__(
         self.ingress_reports = ingress_reports
         self.ingress_reports_uuid = ingress_reports_uuid
 
-        self.metadata_start_date: str = self._context.get("metadata_start_date")
-        self.metadata_end_date: str = self._context.get("metadata_end_date")
+        self.split_files = [Path(file) for file in self._context.get("split_files") or []]
+        self.ocp_files_to_process: dict = self._context.get("ocp_files_to_process")
```

Comment:
Type hints for mappings should specify the [typep of keys and values](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#useful-built-in-types).
```suggestion
        self.ocp_files_to_process: dict[str, dict[str, str]] = self._context.get("ocp_files_to_process")
```